### ISSUE TITLE
Add Amazon PL to Marketplaces Enum

### ISSUE THREAD

**Issue by ceijssen** [python-amazon-mws/python-amazon-mws#254]
Hi again! This time Amazon has expanded their business to Poland. Can you add these credentials to [Marketplaces](https://github.com/python-amazon-mws/python-amazon-mws/blob/e8590b5fc0685114db2676e3179217a27efa280d/mws/mws.py#L41) to support it? It's not yet listed in https://docs.developer.amazonservices.com/en_US/dev_guide/DG_Endpoints.html though it should appear there soon because they are [onboarding sellers already](https://amazoneu.au1.qualtrics.com/jfe/form/SV_0N8B0FR0ecRX81v).

Amazon MWS Endpoint: mws-eu.amazonservices.com
PL Marketplace ID: A1C3SOZRARQ6R3

Thank you :)

**Comment by GriceTurrble** [comment]
Thank you for the update.

I'll check in and add that to the Enum once they have it listed in their docs.

For the time being, you can do the following without waiting on an update from us:

1. Ensure the endpoint is set properly by assigning the `domain` attribute on any API class:

   ```python
   from mws import Feeds
   feeds_api = Feeds(...)
   # The default region is 'US', but its only effect is to set the domain,
   # which you can override easily:
   feeds_api.domain = "https://mws-eu.amazonservices.com"
   ```

2. Use the MarketplaceId string literal in place of a `marketplace_id` in any request that needs it:

   ```python
   PL_MARKETPLACE_ID = "A1C3SOZRARQ6R3"
   
   feeds_api.submit_feed(
       marketplace_ids=[PL_MARKETPLACE_ID],
       ...
   )
   ```

I'll keep this issue open as a reminder until MWS updates their documentation (assuming they do at some point).

Thank you.

**Comment by GriceTurrble** [comment]
I still don't see Poland listed in the endpoints on their documentation. Then again, MWS is legacy now, so they may not be updating them regularly.

Will check this out and push an update shortly.

**Comment by ceijssen** [comment]
@GriceTurrble thank you. I have been able to push listings to my PL seller account with your suggested code, so I can confirm that the marketplace ID is correct. We've reached out to our partner manager at Amazon NL and he replied they'll add it to the docs "in a next sprint", so that could be any time :sweat_smile:

**Comment by ceijssen** [comment]
@GriceTurrble Poland is added to the docs https://docs.developer.amazonservices.com/en_US/dev_guide/DG_Endpoints.html

**Comment by Bobspadger** [comment]
@ceijssen @GriceTurrble  I'll sort this today and prep a release for PyPI

**Comment by Bobspadger** [comment]
Both PR's are in, I'll let @GriceTurrble double check them, then I can merge and createa pypi release.

**Comment by Bobspadger** [comment]
https://pypi.org/project/mws/0.8.12/

I've just released 0.8.12 to pypi - enjoy!
